src/ol_infrastructure/applications/mitxpro/__main__.py

```diff
@@ -8,74 +7,27 @@
     OLVaultDatabaseBackend,
     OLVaultMariaDatabaseConfig,
 )
-from ol_infrastructure.infrastructure.operations.consul import (
-    consul_server_security_group,
-)
-from ol_infrastructure.lib.ol_types import AWSBase
+from ol_infrastructure.lib.ol_types import Apps, AWSBase
 from ol_infrastructure.lib.stack_defaults import defaults
 
 stack = get_stack()
 stack_name = stack.split(".")[-1]
 namespace = stack.rsplit(".", 1)[0]
 env_suffix = stack_name.lower()
 network_stack = StackReference(f"infrastructure.aws.network.{stack_name}")
+consul_stack = StackReference("infrastructure.operations.consul")
 dns_stack = StackReference("infrastructure.aws.dns")
 mitodl_zone_id = dns_stack.require_output("odl_zone_id")
 mitxpro_vpc = network_stack.require_output("mitxpro_vpc")
 operations_vpc = network_stack.require_output("operations_vpc")
 mitxpro_environment = f"mitxpro-{env_suffix}"
+mitxpro_config = Config("mitxpro_edxapp")
 aws_config = AWSBase(
-    tags={"OU": "mitxpro", "Environment": mitxpro_environment},
-)
-
-mitxpro_edxapp_security_group = ec2.SecurityGroup(
-    f"mitxpro-edxapp-access-{env_suffix}",
-    name=f"mitxpro-edxapp-access-{env_suffix}",
-    description="Access control to mitxpro_edxapp",
-    ingress=[
-        ec2.SecurityGroupIngressArgs(
-            cidr_blocks=[mitxpro_vpc["cidr"]],
-            ipv6_cidr_blocks=[mitxpro_vpc["cidr_v6"]],
-            protocol="tcp",
-            from_port=22,
-            to_port=22,
-            description="mitxpro_vpc ssh access",
-        ),
-        ec2.SecurityGroupIngressArgs(
-            cidr_blocks=[operations_vpc["cidr"]],
-            ipv6_cidr_blocks=[operations_vpc["cidr_v6"]],
-            protocol="tcp",
-            from_port=22,
-            to_port=22,
-            description="operations_vpc ssh access",
-        ),
-        ec2.SecurityGroupIngressArgs(
-            cidr_blocks=["0.0.0.0/0"],
-            ipv6_cidr_blocks=["::/0"],
-            protocol="tcp",
-            from_port=80,
-            to_port=80,
-            description="HTTP access",
-        ),
-        ec2.SecurityGroupIngressArgs(
-            cidr_blocks=["0.0.0.0/0"],
-            ipv6_cidr_blocks=["::/0"],
-            protocol="tcp",
-            from_port=443,
-            to_port=443,
-            description="HTTPS access",
-        ),
-        ec2.SecurityGroupIngressArgs(
-            cidr_blocks=[mitxpro_vpc["cidr"]],
-            ipv6_cidr_blocks=[mitxpro_vpc["cidr_v6"]],
-            protocol="tcp",
-            from_port=18040,
-            to_port=18040,
-            description="Xqueue access",
-        ),
-    ],
-    tags=aws_config.tags,
-    vpc_id=mitxpro_vpc["id"],
+    tags={
+        "OU": "mitxpro",
+        "Environment": mitxpro_environment,
+        "application": Apps.mitxpro_edx.name,
```

Comment:
I'm pretty sure this should be `.value` if you want the string. Otherwise I believe it will return `"mitxpro_edx"`